### PR TITLE
WIP - Add verified spawn emoji for DTS

### PR DESCRIPTION
**Pull request by DrMurx**
## Description

Adds a new DTS `spawn_verified_emoji` which translates the simple boolean of `spawn_verified` to  a ✅ or a ❓ emoji respectively.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context

Emojis are easier to grasp than a literal boolean.

## How Has This Been Tested?

By adding the new token to a DTS template

## Wiki Update
- [x] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

### CODE REVIEW

**Comment by darkelement1987** [comment]
Could you please add your changes to the docs? And maybe use ❌ for 'Not verified'

**Comment by DrMurx** [comment]
Sure. 

However, I'd like to develop the whole idea a bit further, will push some more commits to this PR tonight. Documentation will be updated as well.

**Comment by JimBim89** [comment]
My local setup uses all three, using ? for no verified data.

I also have verified_emoji_or_empty and unverified_emoji_or_empty.

So you don't have to have checkpoints all the time.

File: PokeAlarm/Events/MonEvent.py [hunk 1]
```diff
@@ -337,3 +333,20 @@ def generate_dts(self, locale, timezone, units):
                 and self.weight <= 2.41 else '')
         })
         return dts
+
+    def _disappear_time_infos(self, time, spawn_verified):
```
Comment (LegitDongo) [inline PokeAlarm/Events/MonEvent.py:337]: This seems like something that would do better in the PokeAlarm.Utils file

File: PokeAlarm/Events/MonEvent.py [hunk 2]
```diff
@@ -143,23 +143,19 @@ def generate_dts(self, locale, timezone, units):
...
-            'time_left': time[0],
-            '12h_time': time[1],
-            '24h_time': time[2],
-            'time_left_no_secs': time[3],
-            '12h_time_no_secs': time[4],
-            '24h_time_no_secs': time[5],
-            'time_left_raw_hours': time[6],
-            'time_left_raw_minutes': time[7],
-            'time_left_raw_seconds': time[8],
-
```
Comment (LegitDongo) [inline PokeAlarm/Events/MonEvent.py:158]: Why were the above lines removed?
I think this is a standard part of dev now

**Review by LegitDongo — CHANGES_REQUESTED** [review]
I like this, just a few questions

File: PokeAlarm/Events/MonEvent.py (hunk 2 above)
Comment (DrMurx) [reply PokeAlarm/Events/MonEvent.py:158]: I accidentally pushed a preliminary commit, that's why I marked the whole PR as WIP. It's seriously broken now, but I'm super-busy atm and will push the full changeset later.

Comment (DrMurx) [reply PokeAlarm/Events/MonEvent.py:158]: As an outline, all the time fields get an `...or_empty` counterpart. Any given despawn time information is just an estimation if the despawn time is not verified, so the template configuration could reflect this.

File: PokeAlarm/Events/MonEvent.py (hunk 1 above)
Comment (DrMurx) [reply PokeAlarm/Events/MonEvent.py:337]: Sure :)

Comment (LegitDongo) [reply PokeAlarm/Events/MonEvent.py:337]: Sorry, I meant PokeAlarm.MonUtils since this will only be used for monsters :smile:

**Comment by LegitDongo** [comment]
@DrMurx what's the status of this?

**Comment by LegitDongo** [comment]
Closed in favor of #710

**Pull request closed without merging**